packages/vats/test/test-distributeFees.js

```diff
@@ -58,12 +63,13 @@ test('fee distribution', async t => {
   const brand = brands.get('moola');
   const { feeDepositFacet, getPayments } = makeFakeFeeDepositFacet(issuer);
   const treasury = makeFakeTreasury();
+  const amm = makeFakeTreasury();
```

Comment:
awkward naming here; I can read it as `makeFakeFeeProducer` or some such, right? (not critical)